oc-chef-pedant/spec/api/keys/keys_spec.rb

```diff
@@ -686,6 +686,40 @@ def get_client_key(org, client, requestor, key)
       platform.delete_org(other_org_name)
     end
 
+    context "when a client exists with the same name as the user" do
+      let(:colliding_client) do
+        {
+          "name" => org_user_name,
+          "public_key" => keys[:org_client][:public],
+          "private_key" => keys[:org_client][:private]
+        }
+      end
+
+      let(:colliding_client_payload) do
+        { "name" => colliding_client['name'], "public_key" => colliding_client['public_key'],
+          "admin" => "true"
```

Comment:
The "admin" here here is a bit of a no-op (admin status is now handled via group members), but having it here and set to "true" is a bit confusing. If this client /were/ an admin, your test would pass regardless of which requestor made the call since admins have READ access on users within their orgs.  Which is a long way to say, could we set this to false even though it is a noop?